azkaban-web-server/build.gradle

```diff
@@ -66,9 +66,7 @@ dependencies {
   generateRestli('com.linkedin.pegasus:generator:' + pegasusVersion)
   generateRestli('com.linkedin.pegasus:restli-tools:' + pegasusVersion)
 
-  // Needed by Velocity at runtime
```

Comment:
Are they no longer needed? 
This change is not mentioned in the change description. This may make it harder to trace problems later.